src/shared/condition.c

```diff
@@ -898,7 +898,7 @@ static int condition_test_path_is_read_write(Condition *c, char **env) {
         assert(c->parameter);
         assert(c->type == CONDITION_PATH_IS_READ_WRITE);
 
-        return path_is_read_only_fs(c->parameter) <= 0;
+        return path_is_read_only_fs(c->parameter) > 0;
```

Comment:
Hm??? The condition is RW, and the function check RO... Why do you change this??

Reply:
oh，I made a mistake. 
But I find that if the path doesn't exist and not null, ConditionPathIsReadWrite will always return true.

Reply:
if (statvfs(path, &st) < 0)
       return -errno;

if statvfs failed， ConditionPathIsReadWrite will always be true.

Reply:
@yuwata

Reply:
The current logic sounds good for me.